time stamp
int32 data
float64[3] cor_data
